registry/config-example.yml

```diff
@@ -0,0 +1,18 @@
+version: 0.1
+log:
+  fields:
+    service: registry
+storage:
+  cache:
+    blobdescriptor: inmemory
+  filesystem:
+    rootdirectory: /var/lib/registry
+http:
+  addr: :5000
+  headers:
+    X-Content-Type-Options: [nosniff]
+health:
+  storagedriver:
+    enabled: true
+    interval: 10s
+    threshold: 3
```

Comment:
@andriisoldatenko newline here and in the entrypoint?

Reply:
good catch.
i was wondering instead of adding binary to repo just curl it from github releases?